koku/koku/feature_flags.py

```diff
@@ -24,6 +24,10 @@ def fallback_true(feature_name: str, context: dict) -> bool:
     return True
 
 
+def fallback_development_true(feature_name: str, context: dict) -> bool:
+    return context.get("environment") == "development"
```

Comment:
Is it useful to make this case insensitive?
```suggestion
    return context.get("environment", "").lower() == "development"
```